tests/test_backend.py

```diff
@@ -46,96 +45,72 @@ def __call__(self, message, dkim_domain=None, dkim_key=None,
-        super().__init__(*args, **kwargs)
-
-        self.connection = boto3.client(
-            'ses',
-            aws_access_key_id='ACCESS_KEY',
-            aws_secret_access_key='SECRET_KEY',
-        )
-
         self.outbox.append(kwargs)
 
     def send_raw_email(self, **kwargs):
         self.outbox.append(kwargs)
-        return {
+        response = {
             'MessageId': 'fake_message_id',
             'ResponseMetadata': {
                 'RequestId': 'fake_request_id',
-            },
-        }
-
-    def send_mail(self, subject, body, from_email, recipient_list):
-        with mock.patch('boto3.client', return_value=self.connection):
-            with mock.patch.object(self.connection, 'send_raw_email', return_value=self.send_raw_email()):
-                # todo not working because we dont call the actual function -> need to override inside send_raw_email
-                send_mail(subject, body, from_email, recipient_list)
-        self.outbox.append(EmailMultiAlternatives(subject, body, from_email, recipient_list))
+                },
+            }
+        return response
 
 
 class FakeSESBackend(django_ses.SESBackend):
-    """
+    '''
     A fake SES backend for testing purposes. It overrides the real SESBackend's
     get_rate_limit method so we can run tests without valid AWS credentials.
-    """
+    '''
 
     def get_rate_limit(self):
         return 10
 
 
-class SESBackendBaseTest(TestCase):
-
-    @classmethod
-    def setUpTestData(cls):
-        super().setUpTestData()
-
-        cls.fake_connection = FakeSESConnection()
-
-        cls.connection = boto3.client(
-            'ses',
-            aws_access_key_id='ACCESS_KEY',
-            aws_secret_access_key='SECRET_KEY',
-        )
+class SESBackendTest(TestCase):
+    def setUp(self):
+        # TODO: Fix this -- this is going to cause side effects
+        django_settings.EMAIL_BACKEND = 'tests.test_backend.FakeSESBackend'
+        django_ses.boto3.client = FakeSESConnection
+        self.outbox = FakeSESConnection.outbox
 
     def tearDown(self):
-        # Empty outbox every time test finishes
-        self.fake_connection.outbox = []
+        # Empty outbox everytime test finishes
+        FakeSESConnection.outbox = []
 
-
-class SESBackendTest(SESBackendBaseTest):
-
-    @staticmethod
-    def _rfc2047_helper(value_to_encode):
+    def _rfc2047_helper(self, value_to_encode):
         # references: https://docs.python.org/3/library/email.header.html, https://tools.ietf.org/html/rfc2047.html
         name, addr = email.utils.parseaddr(value_to_encode)
         name = email.header.Header(name).encode()
         return email.utils.formataddr((name, addr))
 
     def test_rfc2047_helper(self):
         # Ensures that the underlying email.header library code is encoding as expected, using known values
-        unicode_from_addr = u'Unicode Name Österreich <from@example.com>'
+        unicode_from_addr = u'Unicode Name óóóóóó <from@example.com>'
```

Comment:
i chose an actual word with a non-default char, thought it might be more clear what is happening here

Reply:
For the tests I started fresh from the repo master branch, this is why some of your changes were not reflected. I updated the obvious ones you mentioned, but for this one I am not sure. I would rather make the minimal changes to the tests, especially when this change result on different behaviour :

```
AssertionError: '=?utf-8?q?Unicode_Name_=C3=96sterreich?= <from@example.com>' != '=?utf-8?b?VW5pY29kZSBOYW1lIMOzw7PDs8Ozw7PDsw==?= <from@example.com>'
- =?utf-8?q?Unicode_Name_=C3=96sterreich?= <from@example.com>
+ =?utf-8?b?VW5pY29kZSBOYW1lIMOzw7PDs8Ozw7PDsw==?= <from@example.com>
```

You can see that before, the encoding resulting of the tested string was base64 (`VW5pY29kZSBOYW1lIMOzw7PDs8Ozw7PDsw==`), and after your change it's something else (`=C3=`).